code/common/subscriptions.q

```diff
@@ -3,29 +3,31 @@
 \d .sub
 
 AUTORECONNECT:@[value;`AUTORECONNECT;1b];				//resubscribe to processes when they come back up
-checksubscriptionperiod:@[value;`checksubscribtionperiod;0D00:00:10];	//how frequently you recheck connections.  0D = never
+if[@[value;`.proc.lowpowermode;0b];
+  checksubscriptionperiod:@[value;`checksubscribtionperiod;0D00:01:00];	//how frequently you recheck connections.  0D = never
```

Comment:
Use spaces rather than tabs to bring this comment back in a bit.

Reply:
checksubscriptionperiod has a typo - b instead of p

I think it would be better to just turn this off (set to 0D) in low power mode.  Also, you still need a value for non lower power mode.  So the setting could probably be

checksubscriptionperiod:(not @[value;`.proc.lowpowermode;0b]) * @[value;`checksubscriptionperiod;0D00:00:10]

Reply:
line 40 in TorQ/config/settings/default.q defines the checksubscriptionperiod:0D00:00:10. So that would be the default for non low power mode. I don't think it needs to be explicitly defined in subscriptions.q?

Reply:
The standard practice here is to redefine it in the file if it hasn't already been defined, rather than assuming it has already been defined.  That's why all those lines look like @[value.... ]